docs/how_to/embeddings.md

```diff
@@ -0,0 +1,27 @@
+# Embedding support
+
```

Comment:
(for later) this can also benefit from some diagrams

Reply:
totally